Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            const int NumOfPerson = 2;
```

Comment:
Local variables should be camel case. Regardless of being a const or not.

Reply:
It seems like people do things differently, and I must admit, it's a blur to me.  I will keep in mind to do numOfPerson for constant for this class.  I was interested in seeing if Codacy (which I never heard of until this Bootcamp) would suggest any better way of doing it.  I recently saw somebody else wanting NUM_OF_PERSON for constant.  I went to Microsoft site and they did had a "const double GrativationalConstant" at https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/const, and so, I ended up going with NumOfPerson.  I will keep in mind to do camel case for constants.  Thank you.

Reply:
While making suggested fixes, I happen to notice that in Lesson3Tests.cs that came with the original code, there is a a line that has "const string PromptMessage".  However, I went ahead a made change to the constant that I had coded to be camel case. instead.